lib/sidekiq.rb

```diff
@@ -48,7 +47,7 @@ module Sidekiq
     "connected_clients" => "9999",
     "used_memory_human" => "9P",
     "used_memory_peak_human" => "9P"
-  }.freeze
```

Comment:
note this is a behavior change cause Hash is not a string literal so the hash will no longer be frozen

Reply:
In this case, it's not a concern.